view/ViewableData.php

```diff
@@ -541,6 +542,9 @@ public function CSSClasses($stopAtClass = 'ViewableData') {
 			$classes[] = $this->template;
 		}
 
+		// Strip out namespaces
+		$classes = preg_replace('#.*\\\\#', '', $classes);
```

Comment:
If you mean the code in lines 541 - 543, sure, they might need a regex. But this code still causes all PHP classes and template names to be cut down to their unqualified name only.